src/test/testProfileUpdate.js

```diff
@@ -0,0 +1,23 @@
+import chai, { expect } from 'chai';
+import chaiHttp from "chai-http";
+import { describe, it } from 'mocha';
+import app from "../app";
+import registerMock from './mock/registerMocks'
+import updateMock from '../test/mock/updateProfileMock'
+
+chai.use(chaiHttp);
+
+describe("Phantom API test update profile", () => {
+  it("Should not update user if name is empty ", done=>{
+    chai 
+      .request(app)
+      .patch("/api/v1/auth/updateProfile")
+      .set("Authorization", updateMock.token.oparator)
+      .send(registerMock.nameIsEmpty)
+      .end((err,res)=>{
+        expect(res).to.have.status(400);
+        done();
+      })
+  }); 
+  });
+  
```

Comment:
Request new line

Reply:
You can see it's there. Any other way to add it maybe?